src/pkg/track/track.go

```diff
@@ -27,6 +27,7 @@ func Evt(name string, props ...cliClient.Property) {
 	if disableAnalytics {
 		return
 	}
+	term.Debugf("%s %v", name, props)
```

Comment:
Want to keep this? If so, maybe we should add a prefix

```suggestion
	term.Debugf("tracking event %q: %v", name, props)
```